lib/membrane/video_compositor/scene.ex

```diff
@@ -0,0 +1,161 @@
+defmodule Membrane.VideoCompositor.Scene do
+  @moduledoc """
+  Scene describes positions and transformations of the videos, provided to the video compositor.
+  """
+
+  # alias Membrane.VideoCompositor.Position
+  alias Membrane.VideoCompositor.Scene.ComponentsManager, as: Manager
+  alias Membrane.VideoCompositor.Scene.Element
+  alias Membrane.VideoCompositor.Scene.ElementDescription
+  alias Membrane.VideoCompositor.Scene.Video
+
+  @type error_t :: any()
+  @type id_t :: non_neg_integer()
+  @type component_t :: module()
+  @type components_t :: keyword(component_t())
+
+  @typedoc """
+  Description of the scene.
+  """
+  @type scene_description_t() :: [
+          videos: %{required(id_t()) => ElementDescription.t()},
+          scenes: %{required(atom()) => t()}
+        ]
```

Comment:
IMO `scene_spec` is better name, instead of scene description. What do you think?